backend/src/seeds/index.ts

```diff
@@ -13,6 +13,9 @@ import { userFactory } from './factory/user'
 import { creationFactory } from './factory/creation'
 import { transactionLinkFactory } from './factory/transactionLink'
 import { entities } from '@entity/index'
+import CONFIG from '@/config'
+
+CONFIG.EMAIL = false
```

Comment:
Why is this not in the master.
It's not made by me.

May it came in by conflict solving on my side?